pkg/storage/mvcc_test.go

```diff
@@ -5884,6 +5888,220 @@ func TestMVCCGarbageCollectRangesFailures(t *testing.T) {
 	}
 }
 
+func TestMVCCGarbageCollectClearRange(t *testing.T) {
```

Comment:
Maybe we should just move all of these over to `TestMVCCHistories`, which seems more convenient? Fine with deferring that until stability too.

Reply:
They are all covered by mvcc_test_histories, I removed redundant ones. I'll keep the test with a single basic case, it is much easier to repro or check why certain things didn't work compared to doing it with datadriven.